File: src/main/trellofrontend/src/app/model/board.ts
export class Board {
  constructor(id: number, username: string, boardColor: string) {
    this._id = id;
    this._username = username;
    this._boardColor = boardColor;
  }

  private _id: number;

  get id(): number {
    return this._id;
  }

  set id(value: number) {
    this._id = value;
  }

  private _username: string;

  get username(): string {
    return this._username;
  }

  set username(value: string) {
    this._username = value;
  }

  private _boardColor: string;

  get boardColor(): string {
    return this._boardColor;
  }

  set boardColor(value: string) {
    this._boardColor = value;
  }
}

File: src/main/java/com/alexanderbelldev/trelloclone/Model/Board.java
package com.alexanderbelldev.trelloclone.Model;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

@Data
@Entity
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class Board {

    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    Integer id;
    String username;
    String boardColor;

}
